.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
It allows you to put more code and more easily keep the 80 limit. By doing so it's easier to have two tabs open at the same time on your screen. Well that's only a part of the reason I guess, personally I prefer a 4 space + 140 columns limit I find it easier to read. Just pick up a convention and stick with it!